db/relayer.go

```diff
@@ -36,7 +36,15 @@ func (rp *RelayPacket) BeforeCreate(scope *gorm.Scope) error {
 	return nil
 }
 
-func (rp *RelayPacket) AfterCreate(tx *gorm.DB, cfg config.Configuration) (err error) {
+func (rp *RelayPacket) AfterCreate(tx *gorm.DB) (err error) {
+	cfg, err := config.ParseConfig()
```

Comment:
Great work fixing the cfg arg to the hook.

Reply:
I kinda hate reading the file on every row create but had to do it, for now, will deal better in coming PR's